Platon/backend/app/workspace_system/views.py

```diff
@@ -1160,10 +1188,35 @@ def delete(requester_id, self):
                         try:
                             db.session.delete(requested_workspace)
                             db.session.commit()
-                            FileSystem.delete_ws_files(worksapce_id)
+                            FileSystem.delete_ws_files(worksapce_id) # while testing A.S., I've commented out this part since it creates an error. I've assumed getting an error here while running tests locally is not important. Am I right?
```

Comment:
Special note here. @HUmutOzdemir , can you please check the comment that I've written.

Reply:
The reason for this error is the absence of the main folder of a workspace. To reduce the error of this part, you have to create the initial folder structure of a workspace by using the `initialize_ws` function of `FileSystem` class in the `helpers.py` file of `file_system` module.